src/trulens_eval/trulens_eval/database/__init__.py

```diff
@@ -0,0 +1,19 @@
+# ruff: noqa: E401, E402, F401, F403
+"""
+!!! warning
+    This module is deprecated and will be removed. Use `trulens.core.database`
+    instead.
+"""
+
+from trulens.core.utils import deprecation as deprecation_utils
+
+deprecation_utils.packages_dep_warn("trulens_eval.database")
+
+from trulens.core.database import *
+from trulens.core.database import base
```

Comment:
are lines 13-19 necessary given 12?

Reply:
Yes but it looks like line 12 does nothing. `import *` only does anything if `__all__` is defined in the module we are importing from. Removed line 12.

Reply:
Also adding a note explaining this.